src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -164,6 +170,12 @@ struct EnergyDensity : db::SimpleTag {
   using type = Scalar<DataType>;
   static std::string name() noexcept { return "EnergyDensity"; }
 };
+
+template <size_t Dim, typename Frame, typename DataType>
```

Comment:
please document these tags

Reply:
Thank you for the revision! I added documentation to `RicciTensor` and `WeylElectric`; were those all that you were referring to?

Reply:
yes